### PR TITLE
LINK-2089 | prevent to edit/delete if there is payment cancellation/refund

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Prevent to edit/delete if there is payment cancellation/refund

## Closes
[LINK-2089](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2089)

[LINK-2089]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2089?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr161-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr161-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/signupGroup/utils.ts [hunk 1]
```diff
@@ -409,5 +409,7 @@ export const shouldCreatePayment = (
   ) > 0;
 
 export const canEditSignupGroup = (signupGroup: SignupGroup) =>
-  signupGroup.is_created_by_current_user ||
-  signupGroup.has_contact_person_access;
+  (signupGroup.is_created_by_current_user ||
```
Comment (harriris-vincit) [inline src/domain/signupGroup/utils.ts:412]: There probably can be a case where an individual signup in a group is being cancelled or refunded and a user then attempts to cancel the group. Should this also check if any signup in the group has a cancellation or a refund?

**Comment by terovirtanen** [comment]
**LINKEDREGISTRATIONS-UI branch is deployed to platta: https://linkedregistrations-pr161-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedregistrations-pr161-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/signupGroup/utils.ts (hunk 1 above)
Comment (jorilindell) [reply src/domain/signupGroup/utils.ts:412]: Edit is now prevented also if signup group has at least 1 signup with payment cancellation/refund

Comment (jorilindell) [reply src/domain/signupGroup/utils.ts:412]: I also opened a new PR https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/417 to Linked Component UI repo to implement similar logic

**Review by harriris-vincit — APPROVED** [review]
Looks good now :100:

**Pull request merged**